README.md

```diff
@@ -68,8 +68,23 @@ The endpoints for each hashring will be populated automatically by the controlle
 This configuration should be consumed as a ConfigMap volume by the Thanos receivers.
 
 ## About the `--allow-only-ready-replicas` flag
-By default, upon a scale up, the controller adds all new receiver replicas into the hashring as soon as they are in a _running_ state. However, this means the new replicas will be receiving requests from other replicas in the hashring before they are ready to accept them. Due to the nature of how receiver works, it can take some time until receiver's storage is ready. Depending on your roll out strategy, you might see an increased failure rate in your hashring until enough replicas are in a ready state.
+By default, upon a scale up, the controller adds all new receiver replicas into the hashring as soon as they are in a _running_ state.
+However, this means the new replicas will be receiving requests from other replicas in the hashring before they are ready to accept them.
+Due to the nature of how receiver works, it can take some time until receiver's storage is ready.
+Depending on your roll out strategy, you might see an increased failure rate in your hashring until enough replicas are in a ready state.
 
-An alternative is to use the `--allow-only-ready-replicas`, which modifies this behavior. Instead, upon a scale-up, new replicas are added only after it is confirmed they are ready. This means:
+An alternative is to use the `--allow-only-ready-replicas`, which modifies this behavior.
+Instead, upon a scale-up, new replicas are added only after it is confirmed they are ready.
+This means:
 - Old replicas keep operating with the old hashring, until all new replicas are ready. Once this is true, the hashring is updated to include all replicas in the stateful set
-- New replicas will initially come up with the old hashring configuration. This means they will serve only as a "router" and any requests that they receive will be forwarded to replicas in the old hashring. Once _all_ new receiver replicas are ready, the hashring will be updated to include both old and new replicas.
\ No newline at end of file
+- New replicas will initially come up with the old hashring configuration. This means they will serve only as a "router" and any requests that they receive will be forwarded to replicas in the old hashring. Once _all_ new receiver replicas are ready, the hashring will be updated to include both old and new replicas.
+
+
+## About the `--allow-dynamic-scaling` flag
+By default, the controller does not react to voluntary/involuntary distributions to receiver replicas in the StatefulSet.
+This flag allows the user to enable this behavior.
+When enabled, the controller will react to voluntary/involuntary distributions to receiver replicas in the StatefulSet.
```

Comment:
A minor point, but I would totally drop any reference to voluntary/involuntary disruption, since there's no such concept in the Kubernetes world. We could perhaps just say pod or replica disruptions, even though that is questionable for me as well (Is it a disruption if I voluntarily scale down? 🙃). Not super important though...